### PR TITLE
bump version to 1.21.3

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please lint.

**Comment by jakirkham** [comment]
Just noticed we don't have `singledispatch` for Python 2, which appears [upstream]( https://github.com/dask/distributed/blob/1.21.3/requirements.txt#L14 ). Should we be adding that here?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: distributed-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/d/distributed/distributed-{{ version }}.tar.gz
-  sha256: fc9c3a7b27bd2f76aead3016ff147d80396dfa147e1be92a732d836a99817254
+  sha256: fa37da099f589b673006b95b2bdee91930c5609d1b6d25300a4518df2c4e7be1
 
 build:
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Might need to merge with `master` and reset to `0` as it is is now `1` in `master`.

**Comment by jakirkham** [comment]
One minor comment above. Otherwise LGTM.

**Comment by jakirkham** [comment]
Thanks @mrocklin.

Presumably this handles the tornado 5.0 issues that affect Distributed on Python 2.7?

**Comment by mrocklin** [comment]
Yes

**Pull request merged**